docs/docs/spark-getting-started.md

```diff
@@ -26,17 +26,17 @@ Spark is currently the most feature-rich compute engine for Iceberg operations.
 We recommend you to get started with Spark to understand Iceberg concepts and features with examples.
 You can also view documentations of using Iceberg with other compute engine under the [Multi-Engine Support](../../multi-engine-support.md) page.
 
-## Using Iceberg in Spark 3
+## Using Iceberg in Spark
 
 To use Iceberg in a Spark shell, use the `--packages` option:
 
 ```sh
-spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-3.5_2.12:{{ icebergVersion }}
+spark-shell --packages org.apache.iceberg:iceberg-spark-runtime-4.0_2.13:{{ icebergVersion }}
```

Comment:
We can add a variable `sparkVersion` at https://github.com/apache/iceberg/blob/main/site/mkdocs.yml#L86